data/fh/label/de.json

```diff
@@ -535,204 +535,259 @@
+        "1": "Behandelt Wasserfelder als Korridore und Nichtwasserfelder als Hindernisse."
       },
       "lord-of-chaos": {
-        "1": "Force all enemies to move one hex toward the closest unoccupied corridor hex."
+        "1": "Zwinge alle Feinde, sich ein Hexfeld in Richtung des nächsten unbesetzten Korridorhexfelds zu bewegen."
       },
       "orphan": {
-        "1": "self or glowing orb"
+        "1": "Selbst oder leuchtende Kugel"
       },
       "prince-of-frost": {
-        "1": "One character %game.card.recover% one lost card.",
-        "2": "One character loses one card."
+        "1": "Ein Charakter %game.card.recover% ein Verloren-Karte.",
+        "2": "Ein Charakter verliert eine Karte."
       },
       "program-director": {
-        "1": "to an empty hex closest to focus",
-        "2": "Ally to all monsters this round",
-        "3": "Focus on closest enemy",
-        "4": "to any empty hex closest to focus"
+        "1": "zu einem leeren Hexfeld, das dem Fokus am nächsten liegt",
+        "2": "Verbünde dich in dieser Runde mit allen Monstern",
+        "3": "Fokus auf den nächsten Feind",
+        "4": "zu jedem leeren Hexfeld, das dem Fokus am nächsten liegt"
       },
       "render": {
-        "1": "to farthest carving and destroy it.",
-        "2": "Destroy all adjacent carvings."
+        "1": "bis zum weitesten schnitzen und es zerstören.",
+        "2": "Zerstöre alle angrenzenden Schnitzereien."
       },
       "seeker-of-the-abyss": {
-        "1": "All targets lose one card.",
-        "2": "Create one %game.condition.immobilize% trap in an adjacent empty hex.",
-        "3": "All allies add +1 %game.action.attack% to all their attacks this round."
+        "1": "Alle Ziele verlieren eine Karte.",
+        "2": "Erstelle eine %game.condition.immobilize% Falle in einem angrenzenden freien Hex-Feld.",
+        "3": "Alle Verbündete bekommen +1 %game.action.attack% auf alle ihre Angriffe in dieser Runde."
       },
       "sun-demon": {
         "1": "%game.custom.advantage%"
       },
       "tormentor": {
-        "1": "Create 1-hex hazardous terrain in all featureless hexes adjacent to target, enemies in these hexes suffer",
-        "2": "Create 1-hex icy terrain in all featureless hexes adjacent to targets of the attack"
+        "1": "Erschaffe gefährliches 1-Hex-Gelände in allen an das Ziel angrenzenden,<br> freie Hex-Feldern. Feinde in diesen Hexfeldern leiden darunter",
+        "2": "Erschaffe 1 Hexfeld eisiges Gelände in allen freien Hex-Feldern,<br> die an die Angriffsziele angrenzen"
       },
       "vestige-of-the-imprisoned-god": {
-        "1": "Modify terrain.",
-        "2": "All characters who did not attack suffer trap damage at end of round."
+        "1": "Gelände ändern.",
+        "2": "Alle Charaktere, die nicht angegriffen haben, erleiden am Ende der Runde Fallenschaden."
+      }
+	  "banner-spear": {
```

Comment:
Not sure how the check passed, may have to adjust. But here is a missing `,`. Please check the files beforehand. In meantime I check how I can test for this.

Reply:
Okay, checked it but missed it.